service/history/nDCBranchMgr_mock.go

```diff
@@ -1,4 +1,6 @@
-// Copyright (c) 2019 Uber Technologies, Inc.
+// The MIT License (MIT)
+//
+// Copyright (c) 2017 Uber Technologies, Inc.
```

Comment:
Update the LICENSE file?

Reply:
this is the new license file

Reply:
I mean we should change 2017 -> 2019 in the LICENSE file. Maybe in another diff.